themes/jb/layouts/partials/meta.html

```diff
@@ -1,9 +1,9 @@
 <!-- Meta Ttitles -->
-<title itemprop="name">{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}</title>
-<meta property="og:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta name="twitter:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta itemprop="name" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
-<meta name="application-name" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ .Site.Title }}" />
+<title itemprop="name">{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (print .CurrentSection.Title ` ` .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end }}{{ .Site.Title }}</title>
+<meta property="og:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (print .CurrentSection.Title ` ` .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end}}{{ .Site.Title }}" />
```

Comment:
Also, something that might make your `if` a bit clean for your `print` is to use [`printf`](https://gohugo.io/functions/printf/) instead. With something like this:
```suggestion
<meta property="og:title" content="{{ .Title -}}{{- if .Title }} | {{end}}{{ if ne .Title (printf `%s %v` .CurrentSection.Title .Params.episode) }}{{ .Params.show_name -}}{{- if .Params.show_name }} {{ if .Params.episode }}{{ .Params.episode -}}{{end}} | {{end}}{{end}}{{ .Site.Title }}" />
```

Reply:
@elreydetoda Regarding using `printf`, the only reason I used this block (as is) is because it's used elsewhere for exactly this condition (i.e. for matching LAN and avoiding duplication). To that end, I could switch this, but it would mean you'd have two conditional evaluations in your code that are intended to produce the result.

Reply:
For clarity, I copied it from #74.